addons/project_issue/models/project_issue.py

```diff
@@ -0,0 +1,303 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from datetime import datetime
+from openerp import api, fields, models
+from openerp.exceptions import AccessError
+from openerp.tools.translate import _
+
+
+class ProjectIssue(models.Model):
+    _name = "project.issue"
+    _description = "Project Issue"
+    _inherit = ['mail.thread', 'ir.needaction_mixin']
+    _order = "priority desc, create_date desc"
+    _mail_post_access = 'read'
+
+    def _get_default_partner(self):
+        if 'default_project_id' in self.env.context:
+            project = self.env['project.project'].browse(self.env.context['default_project_id'])
+            if project and project.partner_id:
```

Comment:
No need to check this condition. now project and partner_id is recordset. 
we can directly return project.partner_id.id